### ISSUE TITLE
Handle prime-select unknown

### ISSUE THREAD

**Issue by Gara-Dorta** [andrebrait/prime-indicator#22]
Under certain error conditions `prime-select query` might return `unknown` rather than `nvidia` or `intel`. It'd be nice to have that case included, with a custom icon so that the user knows that something is wrong.

**Comment by andrebrait** [comment]
It was in the plans. I ended up forgetting to add this option (it would be a '?' icon or something like that). I'll mark this as a request for improvement :)

**Comment by Gara-Dorta** [comment]
Thanks, a reproducible way to get the unknown status is to active secure boot.
Which by the way makes the readme a bit misleading, the nvidia drivers do work with secure boot, but only after the modules have been [signed](https://www.kernel.org/doc/html/latest/admin-guide/module-signing.html)

**Comment by andrebrait** [comment]
@Garoe I'll add an icon for Unknown then. Will do as soon as I can. This December is being an unusually busy month for me.

Also, thanks for the information. I knew why the NVIDIA drivers wouldn't work but I had no idea one could sign them!

**Comment by mmhobi7** [comment]
https://github.com/andrebrait/prime-indicator/pull/28
